hashtables/ex1/ex1.js

```diff
@@ -1,5 +1,37 @@
 const get_indices_of_item_weight = (weight, length, limit) =>{
-    //your code here
+    const hash_weights = {}
+    
+    weight.forEach((num, index) => {
+        if(Object.keys(hash_weights)[0] == num){
+            hash_weights[num].push(index)
+        } else{
+            hash_weights[num] = [index]
+        }
+    })
+
+    let sumIndices = []
+
+    for(let weight in hash_weights){
+        let sumToLimit = limit - weight
+
+        if(hash_weights[weight].length > 1){
```

Comment:
A different way of doing this would be 
```js
    let sums = [];
    
    for(const key of Object.keys(table)){
        if(Object.keys(table).includes(`${limit-key}`)){
            table[key].forEach(num => sums.push(num))
        }
    }
```